docs/guides/getting_started.md

```diff
@@ -408,10 +408,30 @@ const links = $('a[href]')
     .map((i, el) => $(el).attr('href'))
     .get();
 
-const ourDomain = 'apify.com';
-const absoluteUrls = links.map(link => new URL(link, ourDomain));
+const {protocol, hostname} = new URL(request.loadedUrl);
+const absoluteUrls = links.map(link => new URL(link, request.loadedUrl));
+
+const sameDomainLinks = absoluteUrls.filter(url => url.protocol === protocol && url.hostname.endsWith(hostname));
+
+// ...
+```
+
+This includes subdomains. In order to pin the entire origin, simply compare the `url.origin` property instead:
```

Comment:
I'm not sure if the meaning of an `origin` should be described more extensively.

Reply:
No need to describe. But please add a link under `origin` to a nice explanation somewhere else 👍

Reply:
Also maybe use `filter` instead of `pin`. Pin is very engineeringish term. And the title says filter.